pkg/project/project.go

```diff
@@ -5,9 +5,16 @@ var (
 	name        = "chart-operator"
 	gitSHA      = "n/a"
 	source      = "https://github.com/giantswarm/chart-operator"
-	version     = "n/a"
```

Comment:
Current version is 0.12.3 but going to bump it to 1.0. As per the flattening docs and Helm 3 is coming soon.